### PR TITLE
Upgrade to .NET 6

### PR DESCRIPTION
**Pull request by jesmith026**

### CODE REVIEW

**Comment by tonysneed** [comment]
@jesmith026 After forking the repo, you need to create a branch with your changes, rather than committing to the master branch in your forked repo. 😄

**Comment by jesmith026** [comment]
> @jesmith026 After forking the repo, you need to create a branch with your changes, rather than committing to the master branch in your forked repo. 😄

Seemed unnecessary since I'm deleting these forks right after the upgrade MR's are done.

**Comment by tonysneed** [comment]
> Seemed unnecessary since I'm deleting these forks right after the upgrade MR's are done.

It helps in case I need to make changes on your fork.

File: src/EventDriven.SchemaRegistry.Abstractions/EventDriven.SchemaRegistry.Abstractions.csproj [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>net5.0</TargetFramework>
+        <TargetFramework>net6.0</TargetFramework>
         <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-        <Version>1.0.0-beta2</Version>
+        <Version>1.0.0-beta3</Version>
```
Comment (tonysneed) [inline src/EventDriven.SchemaRegistry.Abstractions/EventDriven.SchemaRegistry.Abstractions.csproj:6]: Let me take care of the package version bump. I may move it out of beta at this time anyway.

**Comment by tonysneed** [comment]
I also fixed the doc generation. This is how VS 2022 handles it.

```xml
<GenerateDocumentationFile>True</GenerateDocumentationFile>
```

**Pull request merged**